DataConnectors/Netskope/Netskope_API_FunctionApp.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "Netskope",
+    "title": "Netskope",
+    "publisher": "Netskope",
+    "descriptionMarkdown": "The [Netskope Cloud Security Platform](https://www.netskope.com/platform) connector provides the capability to ingest Netskope logs and events into Azure Sentinel. The connector provides visibility into Netskope Platform Events and Alerts in Azure Sentinel to improve monitoring and investigation capabilities.",
+    "additionalRequirementBanner":"These queries are based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **Netskope**. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersnetskope)",
```

Comment:
@chicduong, we didn't see the parser PR or it is not included in this PR, is it missed?

Reply:
my apologies, forgot to submit the PR after pushing changes. https://github.com/Azure/Azure-Sentinel/pull/1332